test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -18,38 +11,101 @@ describe('Deployment driver web UI', () => {
     cy.get('@main').contains('Deployment Steps')
     cy.get('@main').get('div.deployProgress').as('statusArea')
     cy.get('@statusArea').get('button.cancelButton').contains('Cancel Deployment')
+
+    cy.get('.pf-c-brand').should('be.visible')
+    cy.get('.pf-c-alert__title')
+      .contains('You currently have a subscription to Ansible Automation Platform')
+    cy.get('p').contains('To manage or setup new subscription, visit the')
+    cy.get('p > .pf-c-button')
+      .contains('Red Hat Hybrid Cloud Console')
+      .should('have.attr', 'href', 'https://console.redhat.com/')
+
+    // Check the H1 title
+    cy.get('h1')
+      .contains('Ansible Automation Platform Deployment Engine')
+      .should('be.visible') 
```

Comment:
Is this really necessary? Would cypress find an invisible element?

Reply:
@tznamena , I think we need to add this extra step to check the UI element's visibility to the end user. Cypress can find an invisible element.

Reply:
Really, that would be bad if they did. Let's leave it in for now.

Reply:
@tznamena , I can show you we will get some test failures if we add `should('be.visible')` if that element is not visible on UI. :-)